vendor/assets/javascripts/trumbowyg/vendor/plugins/contextbar/trumbowyg.contextbar.js

```diff
@@ -87,6 +88,16 @@
                     rect.width == 0;
                   };
 
+                  var getSaveBoundingClientRect = function(node) {
+                    if (node["getBoundingClientRect"]) {
+                      return node.getBoundingClientRect();
+                    } else if (node.parentNode) {
```

Comment:
should I write this like this?
```suggestion
                    } else if (node["parentNode"]) {
```

Reply:
No, dot notation is preferable in js.